homeassistant/components/sensor/uber.py

```diff
@@ -70,13 +72,18 @@ def __init__(self, sensorType, products, product_id, product):
             time_estimate = self._product.get('time_estimate_seconds', 0)
             self._state = int(time_estimate / 60)
         elif self._sensortype == "price":
-            price_details = self._product['price_details']
-            if price_details['low_estimate'] is None:
-                self._unit_of_measurement = price_details['currency_code']
-                self._state = int(price_details['minimum'])
+            price_details = self._product.get('price_details', {})
+            if price_details is not None:
```

Comment:
`price_details` will only be `None` when explicitly set to null in the API. When `price_details` is not found, it will return the second argument, which is `{}`